### PR TITLE
retry with npm, add some cleanup

### PR DESCRIPTION
**Pull request by jonathanong**
https://github.com/dollarshaveclub/face-web/pull/2980

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,10 +18,8 @@ RUN \
   apk --update add tar && \
 
   #
-  # Install yarn & bower
+  # Install bower
   #
-  echo "Installing yarn version: ${YARN_VERSION}" && \
```
Comment (jonathanong) [inline Dockerfile:24]: node alpine already includes yarn

File: Dockerfile [hunk 2]
```diff
@@ -108,6 +110,7 @@ ONBUILD RUN \
     ~/.npm \
     ~/.yarn \
     /tmp/* \
+    /root/.cache \
```
Comment (jonathanong) [inline Dockerfile:113]: delete yarn cache

File: Dockerfile [hunk 3]
```diff
@@ -87,9 +87,11 @@ ONBUILD RUN \
   # Cleanup app
   #
   rm -rf \
+    /app/.git \
     /app/bower_components \
     /app/node_modules \
     /app/tmp \
+    /app/public \
```
Comment (jonathanong) [inline Dockerfile:94]: delete static assets that are served from `dist/`

File: Dockerfile [hunk 4]
```diff
@@ -87,9 +87,11 @@ ONBUILD RUN \
   # Cleanup app
   #
   rm -rf \
+    /app/.git \
```
Comment (jonathanong) [inline Dockerfile:90]: delete the unnecessary git history

**Review by stefkoenig — APPROVED** [review]
Let's do this!

**Pull request merged**